sources/packages/forms/src/form-definitions/sfaa2024-25.json

```diff
@@ -7555,24 +7547,27 @@
                 {
                   "label": "Enter the estimated amount of B.C. income assistance that you will receive during your study period.",
                   "prefix": "$",
+                  "applyMaskOn": "change",
                   "mask": false,
-                  "spellcheck": true,
                   "tableView": false,
-                  "currency": "USD",
+                  "delimiter": true,
+                  "requireDecimal": false,
                   "inputFormat": "plain",
                   "truncateMultipleSpaces": false,
+                  "validate": {
+                    "required": true,
```

Comment:
This component is not a required field in the DEV(main) version. 

![image](https://github.com/user-attachments/assets/375f12d2-277e-4676-9e83-cb58126fc316)

Reply:
Updated.